ui/src/models/cron-workflows.ts

```diff
@@ -21,6 +21,7 @@ export interface CronWorkflowSpec {
     successfulJobsHistoryLimit?: number;
     failedJobsHistoryLimit?: number;
     timezone?: string;
+    schedules?: string[];
```

Comment:
this makes more sense on line 18 right below `schedule`

Reply:
Fixed in #13369 ([this file](https://github.com/argoproj/argo-workflows/pull/13369/files#diff-4393829fdba44d9c8f01c65f50a8dbe38c3e07f8306563d1701e26f4225a9cc5))